exercises/2/webapp/tests/unit/NewsList.spec.js

```diff
@@ -0,0 +1,65 @@
+import { mount, shallowMount } from "@vue/test-utils";
+import NewsList from "@/components/NewsList.vue";
+import NewsItem from "@/components/NewsItem.vue";
+
+describe("empty", () => {
+  it("render empty state", () => {
+    const wrapper = shallowMount(NewsList, {
+      data() {
```

Comment:
You should rather use props for that instead of the internal data. This is a hint, that NewsList does not provide a proper interface for external use.

You can create a initial-news property in the NewsList and use this initial property in the data method by copying the initial property into a new data property. By this, you can inject news from outside and have the news also inside your data